addons/resupply/config.cpp

```diff
@@ -0,0 +1,183 @@
+#include "script_component.hpp"
+
+class CfgPatches
+{
+    class ADDON
+    {
+        name = QUOTE(COMPONENT);
+        units[] = {"Box_SWS_TeamBox","SWS_Ammo_SupplyPod_Empty","SWS_Ammo_SupplyPod_Basic","SWS_Ammo_SupplyPod_Medical","SWS_Ammo_SupplyPod_Howdy"};
+        weapons[] = {};
+        requiredVersion = REQUIRED_VERSION;
+        requiredAddons[] = {
+            "OPTRE_Misc",
+            "SWS_Weapons",
+            "ace_medical_treatment"};
+        author = QUOTE(AUTHOR);
+        VERSION_CONFIG;
+    };
+};
+
+class CfgEditorCategories {
+	class SWSObjects
+	{
+		displayName = "[SWS] Objects"; 
+	};
+};
+class CfgEditorSubcategories {
+    class SWSResupply {
+        displayName = "Supply Pods & Boxes"; 
+    };
+};
+
+class CfgVehicles {
+    
+    class NATO_Box_Base;
+	class Box_SWS_TeamBox : NATO_Box_Base {
+        displayName = "[SWS] Team Supply Box";
+        dlc = "SWS";
+        author = "Black";
+        icon = "iconCrateWpns";
+
+        maximumLoad = 8000;
+        transportMaxWeapons = 14;
+        transportMaxMagazines = 144;
+        transportMaxBackpacks = 12;
+        
+        transportAmmo = 0;
+        transportRepair = 0;
+        transportFuel = 0;
+        supplyRadius = -1;
+
+        armor = 10000;
+
+        model = "\A3\Weapons_F\Ammoboxes\Supplydrop.p3d";
+        editorPreview = "\A3\EditorPreviews_F\Data\CfgVehicles\B_supplyCrate_F.jpg";
+
+        hiddenSelections[] = {"camo"};
+        hiddenSelectionsTextures[] = {"a3\weapons_f\ammoboxes\data\supplydrop_co.paa"};
+
+        scope = 2;
+        scopeCurator = 2;
+        side = 3;
+
+        editorCategory = "SWSObjects";
+        editorSubcategory = "SWSResupply";
+
+        class TransportMagazines {
+                MAG_XX(OPTRE_60Rnd_762x51_Mag,80);
+                MAG_XX(SWS_Magazine_60Rnd_65x39_Caseless_msbs,40);
+                MAG_XX(SWS_Magazine_60Rnd_65x39_Caseless_msbs_Tracers_Blue,15);
+                MAG_XX(SWS_Magazine_60Rnd_65x39_Caseless,40);
+                MAG_XX(SWS_Magazine_60Rnd_65x39_Caseless_Tracers_Blue,15);
+                MAG_XX(OPTRE_36Rnd_95x40_Mag,20);
+                MAG_XX(OPTRE_60Rnd_5x23mm_Mag,30);
+                MAG_XX(OPTRE_15Rnd_762x51_Mag,20);
+        };
+        class TransportItems {
+            ITEM_XX(ACE_bloodIV,30);
+            ITEM_XX(ACE_elasticBandage,100);
+            ITEM_XX(ACE_quikclot,40);
+        };
+        class TransportWeapons {
+            WEAP_XX(SWS_Weapon_M24RL_D_Loaded,8);
+        };
+    };
+    class OPTRE_Ammo_SupplyPod_Empty;
+    class SWS_Ammo_SupplyPod_Empty : OPTRE_Ammo_SupplyPod_Empty {
```

Comment:
Do you want an empty supply pod to be an option? If not, make this ITEM_META(0); and then use SCOPE(2); in the spots that inherit from it.

If it complains about something with scope = 0 try ITEM_META(1); instead. I've had weird issues with it.

Reply:
An empty supply pod is desired for architect reasons; Gotta have that SWS branding.

Reply:
Gotcha, that's fine then.